clang/lib/CIR/CodeGen/CIRGenExpr.cpp

```diff
@@ -220,14 +220,14 @@ static bool isAAPCS(const TargetInfo &TargetInfo) {
   return TargetInfo.getABI().startswith("aapcs");
 }
 
-Address CIRGenFunction::getAddrOfBitFieldStorage(LValue base, 
+Address CIRGenFunction::getAddrOfBitFieldStorage(LValue base,
                                                  const FieldDecl *field,
                                                  unsigned index,
                                                  unsigned size) {
   if (index == 0)
     return base.getAddress();
 
-  auto loc = getLoc(field->getLocation());  
+  auto loc = getLoc(field->getLocation());
```

Comment:
Cosmetic changes unrelated to the patch should come in other PRs, these generate noise in the review, please avoid it next time around. There's a mode for clang-format that it only apply changes to lines changed in a diff, you might wanna look into that.